SharperUniverse/Core/GameRunner.cs

```diff
@@ -18,6 +19,7 @@ public class GameRunner
         internal List<SharperEntity> Entities { get; set; }
         internal ISharperServer Server { get; set; }
         internal int DeltaMs { get; set; }
+	    internal PersistenceManager PeristenceManager { get; set; }
```

Comment:
Is it all straight in your mind how this will be interacted with during gameplay?